.github/workflows/ci.yml

```diff
@@ -164,7 +164,7 @@ jobs:
           cargo test -vv --features="$FEATURES"
       - name: Test const generics
         if: matrix.rust == 'nightly'
-        run: cargo test -p hdf5-types --features hdf5-types/const_generics
+        run: cargo test -p hdf5-types --features hdf5-types/const-generics
```

Comment:
Now that 1.51 is out, I wonder if we should just set MSRV to 1.51, or wait a bit? Hmm..

Crates like arrayvec now require 1.51 (with support for const generics). ndarray is moving slowly in that direction, it's likely that the next major release will require 1.51 (to happen in the foreseeable future).

Reply:
`ndarray` version 0.15 has 1.49 as their official MSRV. 1.51 is only a couple of months after this. Having the `const-generics` flag only makes sense for two compiler versions...

Reply:
I've removed the flag in a commit, but we can revert this if we want to keep it.